addons/im_livechat/static/src/core/common/thread_model_patch.js

```diff
@@ -56,6 +56,10 @@ patch(Thread.prototype, {
         return this.channel_type === "livechat" || super.isChatChannel;
     },
 
+    get composerHide() {
+        return this.livechat_active === false || this.chatbot?.completed;
```

Comment:
`chatbot` will never be present in this patch. As `chatbot` is added for `embed` patch only.

Reply:
Here the condition for hiding composer would be same as `composerDisabled`. So, instead we should return that only. For better maintainability.